### PR TITLE
[stable/artifactory] fixed support for ingress and existingPVC

### PR DESCRIPTION
**Pull request by jainishshah17**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/artifactory/README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ You can customise other parameters in the same way, by passing them on `helm ins
 $ helm delete --purge artifactory
 ```
 This will completely delete your Artifactory Pro deployment.  
-**IMPORTANT:** This will also delete your data volumes. You will lose all data!
+**IMPORTANT:** This will also delete your data volumes. You will loose all data!
```
Comment (unguiculus) [inline stable/artifactory/README.md]: `lose` was actually correct.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/artifactory/README.md (hunk 1 above)
Comment (jainishshah17) [reply stable/artifactory/README.md]: done

File: stable/artifactory/templates/nginx-deployment.yaml [hunk 2]
```diff
@@ -60,14 +66,23 @@ spec:
...
+          mountPath: "/var/opt/jfrog/nginx/ssl"
+        {{- end }}
         resources:
 {{ toYaml .Values.nginx.resources | indent 10 }}
       volumes:
       - name: nginx-volume
       {{- if .Values.nginx.persistence.enabled }}
         persistentVolumeClaim:
-          claimName: {{ template "artifactory.nginx.fullname" . }}
+          claimName: {{ .Values.artifactory.persistence.existingClaim | default (include "artifactory.nginx.fullname" .) }}
```
Comment (eldada) [inline stable/artifactory/templates/nginx-deployment.yaml]: Change `artifactory.persistence.existingClaim` to `nginx.persistence.existingClaim`

Comment (jainishshah17) [reply stable/artifactory/templates/nginx-deployment.yaml]: Done

**Review by unguiculus — APPROVED** [review]

**Pull request merged**